components/common/listingCart/shared/ListingCartItemDetails.vue

```diff
@@ -3,11 +3,22 @@
     <div class="flex justify-between">
       <div class="flex">
         <div>
+          <BaseMediaItem
+            v-if="nft.animationUrl"
+            class="border border-k-shade image is-48x48"
+            :class="{ 'opacity-50': discarded }"
+            :alt="nft?.name"
+            :animation-src="sanitizeIpfsUrl(nft.animationUrl)"
+            mime-type="text/html"
+            preview
+            is-detail />
```

Comment:
What if the mime-type of nft is vedio or 3d-object ?

Reply:
fixed [f5abf6c](https://github.com/kodadot/nft-gallery/pull/9833/commits/f5abf6c1438c22590001dfb058d244c7a39dab46)